release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3340,6 +3340,28 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.37
+[id="ocp-4-14-37_{context}"]
+=== RHSA-2024:6689 - {product-title} 4.14.37 bug fix and security update
+
+Issued: 19 September 2024
+
+{product-title} release 4.14.37, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6689[RHSA-2024:6689] advisory. No RPM packages exist for this update.
```

Comment:
```suggestion
{product-title} release 4.14.37, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6689[RHSA-2024:6689] advisory. There are no RPM packages for this update.
```

Not sure if you all update it, but iirc this is what we have said in the past for no RPM updates.

Reply:
:heavy_check_mark: